azure-cli-diff-tool/azure_cli_diff_tool/_const.py

```diff
@@ -77,5 +92,8 @@
     "1010": "please change property `{0}` from `{1}` to `{2}` for parameter `{3}` of cmd `{4}`",
     "1011": "please confirm sub group `{0}` added",
     "1012": "please confirm sub group `{0}` removed",
+    "1013": "please remove property `{0}` for sub group `{1}`",
+    "1014": "please add back property `{0}` for sub group `{1}`",
+    "1015": "please change property `{0}` from `{1}` to `{2}` for sub group `{3}`",
```

Comment:
I think name variables will make the code more readable, but this may increase the effort.